<launch>

  <!-- Launch the parking controller on the real robot. Launches safety controller, controller node, cone detection node, and the homography transformer -->
  <!-- Edit this to launch your safety controller you implemented in Lab 3! -->
  <!-- <node pkg="wall_follower" name="safety_controller" exec="safety_controller" output="screen">
  </node> -->

  <node pkg="visual_servoing" name="racing_controller" exec="racing_controller" output="screen">
  	<param name="drive_topic" value="/vesc/low_level/input/navigation" />
    <param name="camera_topic" value="/zed/zed_node/rgb/image_rect_color" />
  </node>

  <!-- <node pkg="visual_servoing" name="lane_detector" exec="lane_detector" output="screen">
  </node> -->

  <!-- <node pkg="image_transport" name="republish" exec="republish" args="raw in:=/cone_debug_img out:=/cone_debug_img_compressed" output="screen"></node> -->

</launch>
